lib/YaST/workarounds.pm

```diff
@@ -39,7 +39,7 @@ sub apply_workaround_poo124652 {
     if (!check_screen($mustmatch, %args)) {
         record_info('poo#124652', 'poo#124652 - gtk glitch not showing dialog window decoration on openQA');
         send_key('shift-f3', wait_screen_change => 1);
-        assert_screen('style-sheet-selection-popup');
+        check_screen('style-sheet-selection-popup', 10);
```

Comment:
they are the same, one function through exception the other not, I think you were lucky with the VR,
or what is the difference?

Reply:
`assert_screen` fails when the needle is not matched. That means the other approach with "maximize/unmaximize" screen will never kick in when the "shift-f3" approach does fail. `check_screen` does not fail the complete test,it just exits and the test module goes on and can handle the situation.

Reply:
I see, the problem is that I cannot see the resizing, could you please put one save_screenshot after both send_(alt-f10...) and run some verification just in case, then we can merge it.

Reply:
feel free to add those screenshot any time. Let's merge this for PublicRC.